tasks/plus/setup-license.yml

```diff
@@ -8,6 +8,8 @@
   copy:
     src: "{{ item }}"
     dest: /etc/ssl/nginx
+    mode: 0640
```

Comment:
```suggestion
```

Limiting permissions for the license causes some issues with the installation process. Better to get rid of it for the time being.

Reply:
Removed for now.